features/networking/service.feature

```diff
@@ -786,18 +786,18 @@ Feature: Service related networking scenarios
     Given I use the "<%= cb.masters[1].name %>" node
     #It should work because its external traffic from another node and destination node has a backend pod on it (ETP=local respected)
     When I run commands on the host:
-      | curl --connect-timeout 5 <%= cb.hostip %>:<%= cb.port %> |
+      | curl --connect-timeout 5 <%= cb.hostip %>:<%= cb.port %> || curl --connect-timeout 5 [<%= cb.hostip %>]:<%= cb.port %>|
```

Comment:
Seems || in middle needs to be \|\|

Reply:
Doesn;'t seem to be an issue though working fine but geez, its fetching IPv4..
```
When I run commands on the host:                                                               # features/step_definitions/node.rb:98
      | curl --connect-timeout 5 <%= cb.master1_ip %>:<%= cb.port %> || curl --connect-timeout 5 [<%= cb.master1_ip %>]:<%= cb.port %> |
      [19:54:32] INFO> Shell Commands: http_proxy=http://proxy-user1:JYgU8qRZV4DY4PXJbxJK@10.0.77.69:3128
      https_proxy=http://proxy-user1:JYgU8qRZV4DY4PXJbxJK@10.0.77.69:3128
      oc debug  --kubeconfig=/home/anusaxen/workdir/anusaxen-anusaxenx/ocp4_admin.kubeconfig -n prj-anusaxen-anusaxenx-984e --image=quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:3c52f2e03ba32621265ca94b29022393ff684bc4f7b52f7059550f2107d68065 node/anurag-n6-zzlpj-master-1 -- chroot /host/ bash -c cd\ \'/tmp/workdir/anusaxen-anusaxenx\''
      'curl\ --connect-timeout\ 5\ 10.0.130.84:32340'
      ''
      'curl\ --connect-timeout\ 5\ \[10.0.130.84\]:32340
        % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                       Dload  Upload   Total   Spent    Left  Speed
100    17  100    17    0     0   2833      0 --:--:-- --:--:-- --:--:--  4250
      Hello OpenShift!
        % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                       Dload  Upload   Total   Spent    Left  Speed
100    17  100    17    0     0   8500      0 --:--:-- --:--:-- --:--:--  8500
      Hello OpenShift!
```